### DISCUSSION TITLE
How to show or visualize the local coordinate to facilitate coding xml in simulator?

### DISCUSSION THREAD

**Discussion by Aristomo** [google-deepmind/mujoco#661] [Asking for Help]
How to show or visualize the local coordinate to facilitate coding xml in simulator?

I am writing xml to create the robot model in simulator. However, I find it is a little troublesome to adjust the initial position and especially the initial posture of my model. If I can see the local coordinate, it will be helpful to adjust the initial quaternion.

Thanks.

**Comment by yuvaltassa** [comment]
In `simulate`, under the Render tab, you can visualize various frames. Please consult this [video](https://mujoco.readthedocs.io/en/latest/programming/samples.html#sasimulate), there is also basic help available with the F1 key.
